### DISCUSSION TITLE
Search projects by description in ProjectManager

### DISCUSSION THREAD

**Discussion by me2beats** [godotengine/godot-proposals#4320] [Editor]
I often use project description field in ProjectSettings to add some notes, this could be used to find a project in ProjectManager 
Now projects filtered only by project name

**Comment by Calinou** [comment]
I'm not sure if this is a good idea, as this could result in a lot of false positives appearing if you happen to be search for a word commonly used in descriptions.

Adding a designated tag system (in the form of a PackedStringArray stored in `project.godot`) is likely a better idea. These could be featured on project lines without requiring hovering with the mouse to appear.

**Reply by Zireael07** [reply]
+1 to tags rather than description
